sycl/test-e2e/AddressSanitizer/common/statistics.cpp

```diff
@@ -0,0 +1,31 @@
+// REQUIRES: linux
+// RUN: %{build} %device_asan_flags -O2 -g -o %t
+// RUN: env SYCL_PREFER_UR=1 %{run} %t 2>&1 | FileCheck %s
```

Comment:
`SYCL_PREFER_UR=1` is not needed anymore.

Reply:
done